pysts/vocab.py

```diff
@@ -56,7 +56,12 @@ def embmatrix(self, emb):
             try:
                 embedding_weights[index, :] = emb.g[word]
             except KeyError:
-                embedding_weights[index, :] = np.random.uniform(-0.25, 0.25, emb.N)  # 0.25 is embedding SD
+                if index == 0:
+                    embedding_weights[index, :] = np.zeros(emb.N)
+                else:
+                    embedding_weights[index, :] = np.random.uniform(-0.25, 0.25, emb.N)  # 0.25 is embedding SD
+
+
```

Comment:
Come to think about it, because of the implemented masking, reverting this should not make any impact on the clasrel results. However using nonzero vectors for nonexistent words is very weird.